tests/wpt/src/lib.rs

```diff
@@ -0,0 +1,395 @@
+//! Integration tests running the Web Platform Tests (WPT) for the `boa_runtime` crate.
+#![allow(unused_crate_dependencies)]
+#![cfg(feature = "wpt")]
```

Comment:
I'm wondering if a feature is the correct approach to gate tests...
I think the Rust book recommends ignoring tests for tests that you don't want to always run, but I suppose we would want to have that functionality available.

The other approach would be to use a compiler flag instead (`#![cfg(wpt)]`), which would make things like `--all-features` still work in CI, with the downside of making the command to run the tests a bit less straightforward (`RUSTFLAGS='--cfg wpt' cargo test -p boa_wpt`).

Reply:
That's great, I didn't know you could do that.

Reply:
If we're going with the simple feature, then you should set the `default-members` field on the workspace `Cargo.toml`, to make `--all-features` work on the CI.